src/index.ts

```diff
@@ -100,6 +99,8 @@ export default class FileOrganizer extends Plugin {
   async handleMediaFile(file: TFile, content: string) {
     const humanReadableFileName = await this.generateNameFromContent(content);
     const fileToMove = await this.createFileFromContent(content);
+    // Store the original file name for logging purposes
+    fileToMove.originalFileName = `${file.basename}.${file.extension}`; // Store the original file name as a property of file
```

Comment:
file is a obsidian native object, adding a custom key is not an idea. rather extend the mov content functoin